frontend/package.json

```diff
@@ -4,9 +4,6 @@
   "private": true,
   "proxy": "http://localhost:3001",
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
If those testing libraries are not for production, better put them inside dev dependencies

Reply:
this library is created automatically  by 'create-react-app'